wasm/pre.js

```diff
@@ -41,6 +44,12 @@ Module.initVologramFunctions = () => {
     Module['frame_normals_sz'] = Module.cwrap(
         'frame_normals_sz', 'number'
     );
+    Module['frame_texture_data_ptr'] = Module.cwrap(
```

Comment:
@patrick-volograms In my code I'm using these data pointers as 'number' (I feel like I read something like that in the emscripten docs) but I notice the other data ptr here has 'array' type. I'm not sure if that matters or what the difference is?